cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -119,6 +119,9 @@
       {% if js_source | length > 0 %}
         s.push( '{{ static('js/routes/' + request.path[1:] + 'index.js') }}' );
       {% endif %}
+      {% if flag_enabled('TURBOLINKS', request) %}
+        s.push( '{{ static('js/routes/on-demand/turbolinks.js') }}' );
```

Comment:
Would we want this in base-common.html, which is both the english and spanish page?